src/visualizr/anitalker/face_sr/face_enhancer.py

```
@@ -66,25 +66,34 @@ def enhancer_generator_no_len(images, method="gfpgan", bg_upsampler="realesrgan"
             model_name = "CodeFormer"
-            url = "https://github.com/sczhou/CodeFormer/releases/download/v0.1.0/codeformer.pth"
+            url = (
+                "https://github.com/sczhou/CodeFormer/releases/download/"
+                + "v0.1.0/codeformer.pth"
+            )
     # ------------------------ set up background upsampler ------------------------
     if bg_upsampler == "realesrgan":
         if not torch.cuda.is_available():  # CPU
             import warnings
 
             warnings.warn(
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)